### PR TITLE
Add Annex: Conquer The World as clone to Command & Conquer

### PR DESCRIPTION
**Pull request by Hiradur**

### CODE REVIEW

**Comment by Hiradur** [comment]
I think I made a mistake with the `content` tag here. Can someone tell me if I have to change its value to `free` or `open` and if `free` means free as in no cost or as in freedom. Same mistake in #399.

File: games/c.yaml [hunk 1]
```diff
@@ -259,6 +263,24 @@
...
+  clones:
+  - name: 'Annex: Conquer The World'
+    repo: http://bitbucket.org/annexctw
+    url: http://annexconquer.com/
+    development: active
+    status: playable
+    lang: [XML]
+    framework: [MegaGlest]
+    license: GPL3
+    content: [CC-BY-SA, CC0]
```
Comment (nikuda) [inline games/c.yaml]:
```yaml
license: [GPL3, CC-BY-SA, CC0]
content: open
```

Comment (nikuda) [inline games/c.yaml]:
```yaml
license: [GPL3, CC-BY-SA, CC0]
content: open
```

**Comment by Hiradur** [comment]
Please don't merge until #406 is resolved.

**Pull request closed without merging**